api/lib/opentelemetry/trace/link.rb

```diff
@@ -22,14 +22,15 @@ class Link
 
       # Returns the frozen attributes for this link.
       #
-      # @return [Hash<String, Object>]
+      # @return [Hash{String => String, Numeric, Boolean}]
       attr_reader :attributes
 
       # Returns a new immutable {Link}.
       #
       # @param [SpanContext] span_context The context of the linked {Span}.
-      # @param [optional Hash<String, Object>] attributes A hash of attributes for
-      #   this link. Attributes will be frozen during Link initialization.
+      # @param [optional Hash{String => String, Numeric, Boolean}] attributes A
+      #   hash of attributes for this event. Attributes will be frozen during
```

Comment:
This should be `link`, not event.

Reply:
Sorry about that -- got a little sloppy with cut-and-paste. Should be fixed now.